### PR TITLE
Pager tblidx

### PR DESCRIPTION
**Pull request by jenswi-linaro**

### CODE REVIEW

File: core/arch/arm/mm/tee_pager.c [hunk 1]
```diff
@@ -515,6 +527,10 @@ void tee_pager_early_init(void)
 		assert(pager_tables[n].tbl_info.level == TBL_LEVEL);
 
 		pager_tables[n].pgt.tbl = pager_tables[n].tbl_info.table;
+		pager_tables[n].pgt.vabase = pager_tables[n].tbl_info.va_base;
+		if (n + 1 < num_pager_tables)
+			SLIST_NEXT(&pager_tables[n].pgt, link) =
+				&pager_tables[n + 1].pgt;
```
Comment (jforissier) [inline core/arch/arm/mm/tee_pager.c:533]: Unrelated change ?

**Review by jforissier — COMMENTED** [review]
- For commit "mk/compile.mk: drop -Waggregate-return warning":
`Reviewed-by: Jerome Forissier <jerome@forissier.org>`
- For commit "core: pager: add struct tblidx":
One comment, with that addressed,
`Acked-by: Jerome Forissier <jerome@forissier.org>`

File: core/arch/arm/mm/tee_pager.c (hunk 1 above)
Comment (jenswi-linaro) [reply core/arch/arm/mm/tee_pager.c:533]: You're right, I'll remove it.

**Comment by jenswi-linaro** [comment]
Comment addressed and tags applied.

**Comment by etienne-lms** [comment]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>`

**Comment by jenswi-linaro** [comment]
Tag applied
